### DISCUSSION TITLE
How do you log out a user?

### DISCUSSION THREAD

**Discussion by burggraf** [pocketbase/pocketbase#11] [Q&A] [answered]
I don't see an auth endpoint to log a user out?

**Comment by burggraf** [comment, marked as answer]
`client.AuthStore.clear();`

It's not super obvious, but I found it.

**Reply by ganigeorgiev** [reply]
Yes, that's the correct one.
It is listed in the Authentication API docs examples section - https://pocketbase.io/docs/api-authentication/, but I'd consider stating it more more clearly somewhere else in the docs.

_Just to clarify, PocketBase doesn't really have session in the traditional way and therefore there is no logout endpoint (even the tokens are not stored in the database). The above call just removes the cached token and model data from the client's auth store._

**Reply by GuiBibeau** [reply]
If I hydrate sessions by reading cookies, would you recommend clearing them at the same time?

**Reply by ganigeorgiev** [reply]
@GuiBibeau It depends on your use case. To sync the AuthStore state you can bind to the `pb.authStore.onChange`:

```js
pb.authStore.onChange((token, model) => {
    console.log('New store data:', token, model)
});
```
More info about the AuthStore you can find in https://github.com/pocketbase/js-sdk#authstore.

For an SSR example flow, you can check https://github.com/pocketbase/js-sdk#ssr-integration.
